### PR TITLE
Added Audioware

### PR DESCRIPTION
**Pull request by donk7413**

### CODE REVIEW

**Comment by donk7413** [comment]
Anyone can check ?

**Review by E1337Kat — APPROVED** [review]

**Comment by E1337Kat** [comment]
caught me at a good timeish. Code looks good, but there are test failures. If you want to fix those, I can add them to the next release.

File: test/unit/mods.example.core.audioware.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        path.join(`r6\\scripts\\Audioware\\Natives.reds`),
+        path.join(`r6\\scripts\\Audioware\\Preset.reds`),
+        path.join(`r6\\scripts\\Audioware\\Service.reds`),
+        path.join(`r6\\scripts\\Audioware\\Settings.reds`),
+        path.join(`r6\\scripts\\Audioware\\System.reds`),
+        path.join(`r6\\scripts\\Audioware\\Tween.reds`),
+        path.join(`r6\\scripts\\Audioware\\Utils.reds`)
+      ],
+      outInstructions: [
+        createdDirectory(`r6\\tweaks\\`), // This is a special case
```
Comment (E1337Kat) [inline test/unit/mods.example.core.audioware.ts:74]:
```suggestion
        createdDirectory(`r6\\audioware\\`), // This is a special case
```

I think this is one of the failures.

**Comment by E1337Kat** [comment]
Ok, got the test fixed, so gonna merge it in. Thanks for this!

**Pull request merged**